pkg/status/templates/collector.tmpl

```diff
@@ -27,12 +27,15 @@ Collector
     {{ $version := version $CheckInstances }}
     {{$CheckName}}{{ if $version }} ({{$version}}){{ end }}
     {{printDashes $CheckName "-"}}{{- if $version }}{{printDashes $version "-"}}---{{ end }}
-    {{- range $CheckInstances }}
+    {{- range $instance := $CheckInstances }}
       Instance ID: {{.CheckID}} {{status .}}
       Configuration Source: {{.CheckConfigSource}}
       Total Runs: {{humanize .TotalRuns}}
       Metric Samples: Last Run: {{humanize .MetricSamples}}, Total: {{humanize .TotalMetricSamples}}
       Events: Last Run: {{humanize .Events}}, Total: {{humanize .TotalEvents}}
+      {{- range $k, $v := .TotalEventPlatformEvents }}
+      {{ $k }}: Last Run: {{humanize (index $instance.EventPlatformEvents $k) }}, Total: {{humanize $v}}
+      {{- end }}
```

Comment:
I did this initially but then decided against it because all the "events" are going to be completely different. So I don't see a good reason why "Database Monitoring Query Samples" should be grouped alongside, for example, "SNMP Events." 

Another reason to keep it separate is that the "Event Platform Event" grouping is more of an internal implementation detail rather than a useful user-facing concept.

Reply:
> Another reason to keep it separate is that the "Event Platform Event" grouping is more of an internal implementation detail rather than a useful user-facing concept.

Those are good points. Make sense then 👍